packages/legend-extension-dsl-text/src/components/studio/TextElementEditor.tsx

```diff
@@ -55,13 +62,22 @@ export const TextElementEditor = observer(() => {
   const textElement = textEditorState.textElement;
   const isReadOnly = textEditorState.isReadOnly;
   const typeNameRef = useRef<HTMLInputElement>(null);
+  const [showPreview, setShowPreview] = useState(
```

Comment:
Or another thought is we could have a Preview state for the editor, and its plugin-based, if the type is something we have support for (for example, `markdown`, we show the preview support), else, we disable it. I'm fine with either approach

Reply:
Ok .. I have created too separate components `renderMarkdownView` & `renderPlainTextEditorView`. 
So depending on the `isMarkdown`, it will display one of the views.